### PR TITLE
ergoCubSN001 - update network file

### PR DESCRIPTION
**Pull request by davidetome**
This PR updates the network file of the robots ergoCub SN001 and ergoCubSN002

cc @valegagge

### CODE REVIEW

File: ergoCubSN001/network.ergoCubSN001.xml [hunk 1]
```diff
@@ -227,130 +286,141 @@
...
+        </board>
+
         <board type='foc' name="foc.7.1">
             <ondevice>ETH</ondevice>
             <ataddress ip="10.0.1.7" canbus="1" canadr="1"  />
             <connected bus="CAN" />
         </board>
 
-        <board type='foc' name="foc.7.2">
+        <board type='foc' name="foc.7`.2">
```
Comment (pattacini) [inline ergoCubSN001/network.ergoCubSN001.xml:339]: There could be a typo here.

**Review by pattacini — CHANGES_REQUESTED** [review]
Hi @davidetome 

Could you provide some insights underlying this big change?
Also, would it be worth covering the other ergoCub as well?

Possibly, there's one typo requiring attention. Didn't spot others but, please, check it.

**Comment by davidetome** [comment]
> Hi @davidetome
> 
> Could you provide some insights underlying this big change? Also, would it be worth covering the other ergoCub as well?
> 
> Possibly, there's one typo requiring attention. Didn't spot others but, please, check it.

Ciao @pattacini !!!
 
   I noticed that the [network ](https://github.com/robotology/robots-configuration/blob/devel/ergoCubSN001/network.ergoCubSN001.xml)file was not aligned to the real [architecture ](https://github.com/icub-tech-iit/electronics-wiring/blob/master/ergocub1/ergocub1.2/pdf/ergoCub1.2_Logic_17929_1.2.0.pdf),

I just fixed some errors and checked again w/ the schematic. 

Also SN002 is not correct (probably copied when the folder was created)

cc @valegagge @AntonioConsilvio

**Comment by pattacini** [comment]
> I noticed that the [network ](https://github.com/robotology/robots-configuration/blob/devel/ergoCubSN001/network.ergoCubSN001.xml)file was not aligned to the real [architecture ](https://github.com/icub-tech-iit/electronics-wiring/blob/master/ergocub1/ergocub1.2/pdf/ergoCub1.2_Logic_17929_1.2.0.pdf),
> 
> I just fixed some errors and checked again w/ the schematic.

Superb! Thanks heaps!


> Also SN002 is not correct (probably copied when the folder was created)

Then, it'd be great if we could fix it up as well.

@AntonioConsilvio, you may consider checking these things out for SN000, which is by us these days.

**Comment by davidetome** [comment]
@pattacini @AntonioConsilvio , as suggested by @valegagge , I also fixed SN002 with the last commit 👍🏼

**Comment by pattacini** [comment]
Thanks @davidetome 🚀

**Review by pattacini — APPROVED** [review]
Awaiting the PR to be put in ready for review.

**Comment by AntonioConsilvio** [comment]
Hi! Well done @davidetome.

I'd just like to add that some boards are actually not connected to each other.

For example, the `10.0.1.2` board, has no boards before it, because it's connected to the ETH switch (Gigablox) so it might make sense to edit line 66 in exactly the same way as 10.0.1.5 (i.e. without the `prev` parameter - line 266)? 
Similarly, 10.0.1.21 has no boards after it, so it might make sense to remove the `next` parameter?

Other boards might have this little problem, in the architecture the ETH switch is represented like this:

image

cc @pattacini

**Comment by davidetome** [comment]
> Hi! Well done @davidetome.
> 
> I'd just like to add that some boards are actually not connected to each other.
> 
> For example, the `10.0.1.2` board, has no boards before it, because it's connected to the ETH switch (Gigablox) so it might make sense to edit line 66 in exactly the same way as 10.0.1.5 (i.e. without the `prev` parameter - line 266)? Similarly, 10.0.1.21 has no boards after it, so it might make sense to remove the `next` parameter (line 56)?
> 
> Other boards might have this little problem, in the [architecture](https://github.com/icub-tech-iit/electronics-wiring-public/blob/master/ergocub1/ergocub1.2/pdf/ergoCub1.2_Logic_17929_1.2.0.pdf) the ETH switch is represented like this:
> 
> image
> 
> cc @pattacini

ciao @AntonioConsilvio , it doesn't matter...the previous and next tag are only for info when reading the file.
you can also not declare them and the script runs  the same

**Comment by davidetome** [comment]
@AntonioConsilvio , just to make you happy I just fixed them with the last commit 😄  👍🏼

**Review by AntonioConsilvio — APPROVED** [review]

**Review by valegagge — APPROVED** [review]
LGTM

**Pull request merged**